### PR TITLE
allow alternate coopr plugins root

### PR DESCRIPTION
**Pull request by dereklwood**
first step towards fixing https://issues.cask.co/browse/COOPR-782
- [x] allow `setup.sh` to use an alternate root for plugin data (ie a freshly cloned git repo)

### CODE REVIEW

File: bin/setup.sh [hunk 1]
```diff
@@ -46,7 +47,7 @@ wait_for_plugin_registration () {
 }
 
 load_bundled_data ( ) {
-  __skriptz=$(ls -1 ${COOPR_HOME}/provisioner/lib/provisioner/worker/plugins/*/*/load-bundled-data.sh 2>&1)
+  __skriptz=$(ls -1 ${COOPR_PROVISIONER_PLUGIN_DIR}/*/*/load-bundled-data.sh 2>&1)
```
Comment (awholegunch) [inline bin/setup.sh:50]: So, these are not the same, right?

Goes from:
 `${COOPR_HOME}/provisioner/lib/provisioner/worker/plugins`

to:
`${COOPR_HOME}/provisioner/lib/worker/plugins`

drops `provisioner` in the middle?

Comment (dereklwood) [inline bin/setup.sh:50]: good catch.  the original definition of `COOPR_PROVISIONER_PLUGIN_DIR` was incorrect (it was defined but never used).  will fix

Comment (dereklwood) [inline bin/setup.sh:50]: fixed

**Comment by awholegunch** [comment]
LGTM

**Pull request merged**